src/patternfly/components/Nav/nav.scss

```
@@ -113,6 +117,8 @@
   // * Nav horizontal subnav
   --#{$nav}--m-horizontal--m-subnav--BackgroundColor: var(--pf-t--global--background--color--secondary--default);
   --#{$nav}--m-horizontal--m-subnav--BorderRadius: var(--pf-t--global--border--radius--pill);
+  --#{$nav}--m-horizontal--m-subnav--BorderWidth: var(--pf-t--global--border--width--divider--default);
+  --#{$nav}--m-horizontal--m-subnav--BorderColor: var(--pf-t--global--border--color--high-contrast--default);
```

Typo in the color var, it isn't showing up currently.

Also can you use a HC border width here? Towards the end of development, we can investigate if there is a pattern for boxes like this that need a border in HC and propose a semantic token if it makes sense (like with the plain action tokens)

```suggestion
  --#{$nav}--m-horizontal--m-subnav--BorderWidth: var(--pf-t--global--border--width--high-contrast--regular);
  --#{$nav}--m-horizontal--m-subnav--BorderColor: var(--pf-t--global--border--color--high-contrast);
```